set.mm

```diff
@@ -215070,8 +215081,8 @@ an empty base set ( ~ str0 ) and any relation partially orders an empty
       YHYCXFCXMYKVGYHXMFUJYLXIVHYIXFCXMFVIVOVJVKVLCDEKUAVMXHXJKWIXGFFHUOLVPVNVQ
       VRVSNWKXHWMXJKWIWJXGBXFCFBXFVHWPFUBOVTWAWBWLXIKBXFCFOWCWFWDWEWG $.
 
-    $( Domain of the least upper bound function of a poset.  (Contributed by
-       NM, 6-Sep-2018.) $)
+    $( Domain of the least upper bound function of a set.  (Contributed by NM,
```

Comment:
Why? "Poset" is good, here and below.

Reply:
The definition doesn’t mention anything about poset. In fact for a poset, there is at most one glb/lub for any subset, proved somewhere in dual iirc. So the uniqueness is unnecessary for a poset. See my eldm2 theorems. I think this theorem works for any sets instead of just posets.

Reply:
The role of comments is not to strictly paraphrase the formal statement (the formal statement suffices for that), but to convey the idea of the statement. "Least upper bound" needs at least some kind of order relation to make sense, even if the encoding in set.mm allows some "junk theorems", so "poset" is good here.

Reply:
That's why I wrote "some kind of order relation". Anyway, I think "poset" is good enough, and then people are free to wonder what happens under weaker hypotheses.

Reply:
https://github.com/metamath/set.mm/pull/4248/commits/96f7ae40303f748973d089deaf8bdf6604ba337d